src/control-plane/backend/lambda/api/model/stacks.ts

```diff
@@ -120,6 +120,7 @@ export class CIngestionServerStack extends JSONObject {
       'EnableAuthentication',
       'AuthenticationSecretArn',
       'EnableApplicationLoadBalancerAccessLog',
+      'Protocol',
```

Comment:
When enabling SSL for an existing pipeline, we need update the custom domain and certification too. Does it support?

Reply:
They already are allowed in this list.